tests/arpack/step-36_ar_with_iterations.cc

```diff
@@ -0,0 +1,373 @@
+/* ---------------------------------------------------------------------
```

Comment:
This file is an exact copy of `tests/arpack/step-36_ar.cc` with the only difference being https://github.com/dealii/dealii/pull/8899/files#diff-20c5b8e0204331b74f322770a09c0407R320. It should be pretty fine to just add this one line to the original test.